.gitignore

```diff
@@ -13,3 +13,6 @@ prisma/generated
 *.db-journal
 *.pdf
 *.code-workspace
+
+# ignore secrets
+*secret*
```

Comment:
this is neccessary to ignore files that are there from another branch on my machine (feat/cdn) that has other yaml files for certs etc.